docs/quick_start.md

```diff
@@ -190,4 +190,4 @@ Tezos.wallet
 
 [boilerplate]: https://github.com/ecadlabs/taquito-boilerplate
 [smart_contract_source]: https://ide.ligolang.org/p/2sVshnZ_Aat5pIuUypIBsQ
-[smart_contract_on_better_call_dev]: https://better-call.dev/kathmandunet/KT1GJ5dUyHiaj7Uuc8gqfsbdv5tTbEH3fiRP/operations
+[smart_contract_on_better_call_dev]: https://better-call.dev/ghostnet/KT1GJ5dUyHiaj7Uuc8gqfsbdv5tTbEH3fiRP/operations
```

Comment:
Smart contract address needs to be updated

Reply:
yes